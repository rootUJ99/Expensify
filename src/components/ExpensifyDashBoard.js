import React from 'react';
import ReactDOM from 'react-dom';
import ExpenseList from './ExpenseList';
import ExpenseListItems from './ExpenseListItems';
import ExpenseListFilters from './ExpenseListFilters';
import ExpenseSummary from './ExpenseSummary';
const ExpensifyDashBoard = () => (
    <div>
        <ExpenseSummary/>
        <ExpenseListFilters/>
        <ExpenseList/>
    </div>
);
export default ExpensifyDashBoard;