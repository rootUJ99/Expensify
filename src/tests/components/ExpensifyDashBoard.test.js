import React from 'react';
import ExpensifyDashBoard from '../../components/ExpensifyDashBoard';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
//testing dashboard component
test('testing dashboard component',()=>{
    const wrapper = shallow(<ExpensifyDashBoard />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});