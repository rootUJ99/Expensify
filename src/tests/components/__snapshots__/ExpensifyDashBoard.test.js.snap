// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing dashboard component 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
